test/spec_1_tests.ts

```diff
@@ -14,7 +14,7 @@ const data = {
 };
 const subject = "subject-x0";
 
-const cloudevent = new CloudEvent({
+let cloudevent = new CloudEvent({
```

Comment:
Don't use a `let`.
Use a `const` and create new `const`s if needed.

Reply:
Doing it this way was the path of least resistance to updating the tests now that the CloudEvent is frozen.  Doing it this way also allows us to test the cloneWith method without having to write a bunch more tests.

Reply:
Agree - this is exercising the `cloneWith` function, so I think is good as-is.

Reply:
Sounds fine.